### PR TITLE
Fix for Issue #8401

### LINKED ISSUE #8401: Issue with "Generalidades del protocolo HTTP": the image's urls don't work
MDN URL: https://developer.mozilla.org/es/docs/Web/HTTP/Overview

#### Did you test this? If so, how?

<details>
<summary>Page report details</summary>

* Folder: `es/web/http/overview`
* MDN URL: https://developer.mozilla.org/es/docs/Web/HTTP/Overview
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/es/web/http/overview/index.html
* Last commit: https://github.com/mdn/translated-content/commit/695a9b3854c1a00e5d28c35973a09c718d9bce8c
* Document last modified: 2022-08-13T23:43:28.000Z

</details>

### LINKED ISSUE #8672: Issue with "Generalidades del protocolo HTTP": (short summary here please)
MDN URL: https://developer.mozilla.org/es/docs/Web/HTTP/Overview

#### Did you test this? If so, how?

<details>
<summary>Page report details</summary>

* Folder: `es/web/http/overview`
* MDN URL: https://developer.mozilla.org/es/docs/Web/HTTP/Overview
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/es/web/http/overview/index.html
* Last commit: https://github.com/mdn/translated-content/commit/695a9b3854c1a00e5d28c35973a09c718d9bce8c
* Document last modified: 2022-08-13T23:43:28.000Z

</details>

Images are broken.

### PR DESCRIPTION
**Pull request by aegisnull**
### Description
This pull request includes the fix for the problem of broken image urls as reported on Issue #8401

### Motivation
This should improve the overall experience of the page by not having broken links on the main content

### Related issues and pull requests
Fixes #8401
Fixes #8672

### CODE REVIEW

**Comment by Graywolf9** [comment]
Hi! @aegisnull thank you so much for your time and contribution. Since the localizated content take the images from en-US content is not necessary to put the images again, it should work only with the changes on the `index.md` file. Could you try it and modify?

**Comment by aegisnull** [comment]
@Graywolf9 you have a valid point, as requested I have instead removed the duplicate files I added and replaced the dead links with a link to the images in the original en-US content.

File: files/es/web/http/overview/index.md [hunk 1]
```diff
@@ -8,23 +8,24 @@ tags:
   - Visión general
 translation_of: Web/HTTP/Overview
 ---
+
 {{HTTPSidebar}}
 
 HTTP, de sus siglas en inglés: "Hypertext Transfer Protocol", es el nombre de un protocolo el cual nos permite realizar una petición de datos y recursos, como pueden ser documentos {{glossary("HTML")}}. Es la base de cualquier intercambio de datos en la Web, y un protocolo de estructura cliente-servidor, esto quiere decir que una petición de datos es iniciada por el elemento que recibirá los datos (el cliente), normalmente un navegador Web. Así, una página web completa resulta de la unión de distintos sub-documentos recibidos, como, por ejemplo: un documento que especifique el estilo de maquetación de la página web ({{glossary("CSS")}}), el texto, las imágenes, vídeos, scripts, etc...
 
-![A Web document is the composition of different resources](https://mdn.mozillademos.org/files/13677/Fetching_a_page.png)
+![A Web document is the composition of different resources](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/fetching_a_page.png)
```
Comment (Graywolf9) [inline files/es/web/http/overview/index.md:16]: The render automatically found the image if it exists in content repo

```suggestion
![A Web document is the composition of different resources](fetching_a_page.png)
```

**Review by Graywolf9 — CHANGES_REQUESTED** [review]

**Comment by Graywolf9** [comment]
Thank you so much for your attention @aegisnull 

I let you an example, what I mean in my initial comment is that the page automatically take the image from the en-US folder

File: files/es/web/http/overview/index.md [hunk 2]
```diff
@@ -8,23 +8,24 @@ tags:
...
+![HTTP as an application layer protocol, on top of TCP (transport layer) and IP (network layer) and below the presentation layer.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/http-layers.png)Diseñado a principios de la década de 1990, {{glossary("HTTP")}} es un protocolo ampliable, que ha ido evolucionando con el tiempo. Es lo que se conoce como un protocolo de la capa de aplicación, y se transmite sobre el protocolo {{glossary("TCP")}}, o el protocolo encriptado {{glossary("TLS")}}, aunque teóricamente podría usarse cualquier otro protocolo fiable. Gracias a que es un protocolo capaz de ampliarse, se usa no solo para transmitir documentos de hipertexto ({{glossary("HTML")}}), si no que además, se usa para transmitir imágenes o vídeos, o enviar datos o contenido a los servidores, como en el caso de los formularios de datos. {{glossary("HTTP")}} puede incluso ser utilizado para transmitir partes de documentos, y actualizar páginas Web en el acto.
```
Comment (Graywolf9) [inline files/es/web/http/overview/index.md:20]:
```suggestion
![HTTP as an application layer protocol, on top of TCP (transport layer) and IP (network layer) and below the presentation layer.](http-layers.png)Diseñado a principios de la década de 1990, {{glossary("HTTP")}} es un protocolo ampliable, que ha ido evolucionando con el tiempo. Es lo que se conoce como un protocolo de la capa de aplicación, y se transmite sobre el protocolo {{glossary("TCP")}}, o el protocolo encriptado {{glossary("TLS")}}, aunque teóricamente podría usarse cualquier otro protocolo fiable. Gracias a que es un protocolo capaz de ampliarse, se usa no solo para transmitir documentos de hipertexto ({{glossary("HTML")}}), si no que además, se usa para transmitir imágenes o vídeos, o enviar datos o contenido a los servidores, como en el caso de los formularios de datos. {{glossary("HTTP")}} puede incluso ser utilizado para transmitir partes de documentos, y actualizar páginas Web en el acto.
```

File: files/es/web/http/overview/index.md [hunk 3]
```diff
@@ -8,23 +8,24 @@ tags:
...
 
 ## Arquitectura de los sistemas basados en HTTP
 
 {{glossary("HTTP")}} es un protocolo basado en el principio de cliente-servidor: las peticiones son enviadas por una entidad: el agente del usuario (o un proxy a petición de uno). La mayoría de las veces el agente del usuario (cliente) es un navegador Web, pero podría ser cualquier otro programa, como por ejemplo un programa-robot, que explore la Web, para adquirir datos de su estructura y contenido para uso de un buscador de Internet.
 
 Cada petición individual se envía a un servidor, el cuál la gestiona y responde. Entre cada _petición_ y _respuesta_, hay varios intermediarios, normalmente denominados {{glossary("Proxy_server", "proxies")}}, los cuales realizan distintas funciones, como: gateways o {{glossary("Cache", "caches")}}.
 
-![Client server chain](https://mdn.mozillademos.org/files/13679/Client-server-chain.png)
+![Client server chain](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/client-server-chain.png)
```
Comment (Graywolf9) [inline files/es/web/http/overview/index.md:28]:
```suggestion
![Client server chain](client-server-chain.png)
```

File: files/es/web/http/overview/index.md [hunk 4]
```diff
@@ -137,7 +136,7 @@ Existen dos tipos de mensajes HTTP: peticiones y respuestas, cada uno sigue su p
 
 Un ejemplo de petición HTTP:
 
-![A basic HTTP request](https://mdn.mozillademos.org/files/13687/HTTP_Request.png)
+![A basic HTTP request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/http_request.png)
```
Comment (Graywolf9) [inline files/es/web/http/overview/index.md:139]:
```suggestion
![A basic HTTP request](http_request.png)
```

File: files/es/web/http/overview/index.md [hunk 5]
```diff
@@ -151,7 +150,7 @@ Una petición de HTTP, está formado por los siguientes campos:
 
 Un ejemplo de repuesta:
 
-![](https://mdn.mozillademos.org/files/13691/HTTP_Response.png)
+![](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/http_response.png)
```
Comment (Graywolf9) [inline files/es/web/http/overview/index.md:153]:
```suggestion
![](http_response.png)
```

**Review by Graywolf9 — APPROVED** [review]

**Comment by Graywolf9** [comment]
Thank you so much for your time and contribution @aegisnull 

I apply the changes to the urls to keep the short way to use, this to improve de load time and follow the guidelines.

**Pull request merged**